src/setup/platformSetup/index.website.js

```diff
@@ -15,7 +14,8 @@ import Visibility from '../../libs/Visibility';
  * then refresh. If the page is visibile, prompt the user to refresh.
  */
 function webUpdate() {
-    HttpUtils.download('version.json')
+    fetch('/version.json', {cache: 'no-cache'})
+        .then(response => response.json())
```

Comment:
Should we check that the response is ok before trying to return json? Something similar to what we do [here](https://github.com/Expensify/App/blob/d89a148655e9d2b99205bae8f6155c94a41e5917/src/libs/HttpUtils.js#L33-L36). Otherwise we might get errors like the one below:

![Screen Shot 2022-03-30 at 10 45 05 AM](https://user-images.githubusercontent.com/22219519/160911242-fcc6595d-d325-448a-b7a1-d1d1710b7506.png)

Reply:
Sorry but it doesn't make sense to me 

We either check for `ok` and do nothing when it's not, which results only in skipping a valid error log in the console
Or we check for `ok` and throw an error ourselves

If we're not going to do something (in webUpdate) when the response is not `ok` it doesn't make sense to add handling
We'll at least see an error message when a request fails for some reason

If we want to do something when an error happens, we should add a `catch` block - it would capture any error, but I just don't see what we could do in such case - capture a server log "update check failed?"

Reply:
We didn't have any error handling for the error that would be thrown here https://github.com/Expensify/App/blob/d89a148655e9d2b99205bae8f6155c94a41e5917/src/libs/HttpUtils.js#L33-L36

So we don't have any error handling now as well

Reply:
Makes sense! Thanks for clarifying!